components/plugin_marketplace/marketplace_modal.scss

```diff
@@ -135,9 +161,34 @@
         background: transparent;
     }
 
+    .btn-toggle {
+        top: 1rem;
+    }
+
+    .plugin-status-label {
+        position: absolute;
+        top: 5px;
+        right: 10px;
+    }
+
+    .more-modal__actions {
+        position: relative;
+
+        .btn-primary,
+        .btn-outline {
+            position: absolute;
+            right: 120px;
+        }
+
+        .btn-toggle {
+            position: absolute;
+            right: 80px;
+        }
+    }
```

Comment:
This styling would ideally not be using `position: absolute`. I may need some help with this from a UI expert

Reply:
With these adjustments I think you'll get what you want:

diff --git a/components/plugin_marketplace/marketplace_modal.scss b/components/plugin_marketplace/marketplace_modal.scss
index 4f101bccf..6f28a3f91 100644
--- a/components/plugin_marketplace/marketplace_modal.scss
+++ b/components/plugin_marketplace/marketplace_modal.scss
@@ -161,28 +161,17 @@
```diff
         background: transparent;
     }

-    .btn-toggle {
-        top: 1rem;
-    }

     .plugin-status-label {
-        position: absolute;
-        top: 5px;
-        right: 10px;
+        font-size: 12px;
     }

     .more-modal__actions {
-        position: relative;
-
-        .btn-primary,
-        .btn-outline {
-            position: absolute;
-            right: 120px;
-        }
+        display: flex;
+        align-items: center;

         .btn-toggle {
-            position: absolute;
-            right: 80px;
+            margin: 0 8px 0 24px;
         }
     }
```